src/view/navigation/routes.ts

```diff
@@ -45,15 +48,18 @@ export enum Route {
   FARMER = 'Farmer',
   TRANSACTION_LOGS = 'Transaction Log',
   WAREHOUSE = 'Warehouse',
-  SEARCH_FARMER = 'SearchFarmers',
-  ADD_MILK_ENTRY = 'AddMilkEntry',
-  MILK_ENTRY_DETAILS = 'MilkEntryDetails',
-  EDIT_MILK_ENTRY = 'EditMilkEntry',
-  EDIT_FARMER = 'EditFarmer',
-  ADD_FARMER = 'AddFarmer',
-  ADD_EXPORT_ENTRY = 'AddDeliveryEntry',
-  EXPORT_ENTRY_DETAILS = 'DeliveryEntryDetails',
-  EDIT_EXPORT_ENTRY = 'EditDeliveryEntry',
+  SEARCH_FARMER = 'Search Farmers',
+  ADD_MILK_ENTRY = 'Add Milk Entry',
+  MILK_ENTRY_DETAILS = 'Milk Entry Details',
+  EDIT_MILK_ENTRY = 'Edit Milk Entry',
+  ADD_LOAN_ENTRY = 'Add Loan Entry',
+  LOAN_ENTRY_DETAILS = 'Loan Entry Details',
+  EDIT_LOAN_ENTRY = 'Edit Loan Entry',
+  EDIT_FARMER = 'Edit Farmer',
+  ADD_FARMER = 'Add Farmer',
+  ADD_EXPORT_ENTRY = 'Add Delivery Entry',
+  EXPORT_ENTRY_DETAILS = 'Delivery Entry Details',
+  EDIT_EXPORT_ENTRY = 'Edit Delivery Entry',
```

Comment:
I'm pretty sure I edited all of these a while back, did you merger master into your branch?

Reply:
None of this make sense. Idk what the hell is happening.

Reply:
Do you mean back to no spaces between the words?